cargo-near-build/src/near/docker_build/warn_versions_upgrades.rs

```diff
@@ -0,0 +1,198 @@
+use cargo_metadata::DependencyKind;
+use colored::Colorize;
+
+use crate::types::{
+    cargo::metadata::CrateMetadata, near::docker_build::metadata::ReproducibleBuild,
+};
+use std::{str::FromStr, time::Duration};
+
+const MIN_CUTOFF: semver::Version = semver::Version::new(5, 2, 0);
+
+// TODO #A: update to 5.12.0 after near-sdk-rs   pr release
```

Comment:
todo: version is equal to one , where https://github.com/near/near-sdk-rs/pull/1344 is released